### PR TITLE
Multiple admins, mark not registered users

### PR DESCRIPTION
**Pull request by KenG98**
Admin page now marks which users have registered with MLH and which users haven't. The website also now allows multiple administrator emails as such:

* in the .env instead of 
ADMIN_EMAIL=email1
* write
ADMIN_EMAIL=email1,email2,email3

This way we can all have access to the admin panel and accept/reject applicants.

### CODE REVIEW

**Review by lkwatson — APPROVED** [review]
All looks good, thank you Ken. 

@agoncharova or @a--hoang can you review and merge.

File: bin/www [hunk 1]
```diff
@@ -9,6 +9,12 @@ const debug = require('debug')('cloud-client:server')
 const http = require('http')
 
 /**
+ * Separate admin emails into a list of emails.
```
Comment (anna1go) [inline bin/www]: Use // for single line comments

File: bin/www [hunk 2]
```diff
@@ -9,6 +9,12 @@ const debug = require('debug')('cloud-client:server')
...
+ */
+
+process.env.ADMIN_EMAILS = process.env.ADMIN_EMAIL.split(',')
+
```
Comment (anna1go) [inline bin/www]: 👾👾 (space invader; :nit: too much space)

File: views/pages/admin-console.ejs [hunk 3]
```diff
@@ -46,15 +46,17 @@
       </thead>
       <tbody>
         <tr v-for="user in users">
-          <td v-if='user.status != "Not Attending"' style="min-width:150px">
+          <td v-if='user.status != "Not Attending" && user.local.registered != false' style="min-width:150px">
```
Comment (anna1go) [inline views/pages/admin-console.ejs]: We could probably avoid using the double negative:
`!= false -> == true`

**Review by anna1go — CHANGES_REQUESTED** [review]
Added a couple of comments, mostly nitpicks. Thanks for doing this!

**Pull request merged**